linked_list/linked_list.py

```diff
@@ -23,13 +23,60 @@ def __init__(self):
     self.tail = None
 
   def add_to_tail(self, value):
-    pass
+    #create a new node
+    node = Node(value) 
+    # if the LL is not empty
+    if self.tail is not None:
+      #then set the tail's next to the new node
+      self.tail.set_next(node)
+    else:
+      #if it is empty set the new node to the head
+      self.head = node
+      #set the LL's tail to the new node
+    self.tail = node
+
 
   def remove_head(self):
-    pass
+
+    if self.head == None and self.tail == None:
+      return None
+
```

Comment:
Your code is very clean Austin, and great job with the linked lists! Awesome job